icepyx/core/icesat2data.py

```diff
@@ -1,843 +1,11 @@
-        **kwargs : key-value pairs
-            Additional parameters to be passed to the subsetter.
-            By default temporal and spatial subset keys are passed.
-            Acceptable key values are ['format','projection','projection_parameters','Coverage'].
-            The variable 'Coverage' list should be constructed using the `order_vars.wanted` attribute of the object.
-            At this time (2020-05), only variable ('Coverage') parameters will be automatically formatted.
-
-        See Also
-        --------
-        granules.download
-        """
-        """
-        extract : boolean, default False
-            Unzip the downloaded granules.
-
-        Examples
-        --------
-        >>> reg_a = icepyx.icesat2data.Icesat2Data('ATL06',[-55, 68, -48, 71],['2019-02-20','2019-02-28'])
-        >>> reg_a.earthdata_login(user_id,user_email)
-        Earthdata Login password:  ········
-        >>> reg_a.download_granules('/path/to/download/folder')
-        Beginning download of zipped output...
-        Data request [##########] of x order(s) is complete.
-        """
-
-        # if not os.path.exists(path):
-        #     os.mkdir(path)
-        # os.chdir(path)
-
-        if not hasattr(self, "_granules"):
-            self.granules
-
-        if restart == True:
-            pass
-        else:
-            if (
-                not hasattr(self._granules, "orderIDs")
-                or len(self._granules.orderIDs) == 0
-            ):
-                self.order_granules(verbose=verbose, subset=subset, **kwargs)
-
-        self._granules.download(verbose, path, session=self._session, restart=restart)
-
-    # DevGoal: add testing? What do we test, and how, given this is a visualization.
-    # DevGoal(long term): modify this to accept additional inputs, etc.
-    # DevGoal: move this to it's own module for visualizing, etc.
-    # DevGoal: see Amy's data access notebook for a zoomed in map - implement here?
-    def visualize_spatial_extent(
-        self,
-    ):  # additional args, basemap, zoom level, cmap, export
-        """
-        Creates a map displaying the input spatial extent
-
-        Examples
-        --------
-        >>> icepyx.icesat2data.Icesat2Data('ATL06','path/spatialfile.shp',['2019-02-22','2019-02-28'])
-        >>> reg_a.visualize_spatial_extent
-        [visual map output]
-        """
+    def __init__(self,):
 
-        world = gpd.read_file(gpd.datasets.get_path("naturalearth_lowres"))
-        f, ax = plt.subplots(1, figsize=(12, 6))
-        world.plot(ax=ax, facecolor="lightgray", edgecolor="gray")
-        geospatial.geodataframe(self.extent_type, self._spat_extent).plot(
-            ax=ax, color="#FF8C00", alpha=0.7
+        warnings.filterwarnings("always")
+        warnings.warn(
+            "Depricated. Please use icepyx.Query to create a download data object (all other functionality is the same)",
+            DepricationWarning,
```

Comment:
Do you mean `DeprecationWarning`?

https://docs.python.org/3/library/exceptions.html#DeprecationWarning

Also, I've seen these warnings usually in all caps:

 "DEPRECATED. Please use ..."

Reply:
Yes, I do mean DeprecationWarning... apparently spelling is hard!